noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt

```diff
@@ -67,21 +67,14 @@ val Context.currentConnectivityState: ConnectionState
     }
 
 private fun getCurrentConnectivityState(connectivityManager: ConnectivityManager): ConnectionState {
-    var currentState: ConnectionState = ConnectionState.Unavailable
-
     // Retrieve current status of connectivity
-    connectivityManager.allNetworks.forEach { network ->
-        val networkCapability = connectivityManager.getNetworkCapabilities(network)
-
-        networkCapability?.let {
-            if (it.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)) {
-                currentState = ConnectionState.Available
-                return@forEach
-            }
-        }
+    val connected = connectivityManager.allNetworks.any { network ->
```

Comment:
[`any`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/any.html) is just a really precise way of what the logic in this function is meant to do.

Reply:
Looks good!